specs/ClientConnectionMigrationWhenIngressReload.md

```diff
@@ -0,0 +1,67 @@
+# Connection migration when ingress reload
+
+The feature supports the migration of client connection to another server connection on the same app server whenever feasible, in order to prevent unnecessary disconnections.
+
+## Basic Flow
+
+![image](images/ingress-reload/overview.png)
+
+- Ingress A is being reloaded.
+- Client connection A connects to Ingress B.
```

Comment:
Client connection A connects through Ingress B to server connection A
Client connection B connects through Ingress A to server connection A